app/models/show_api_action.rb

```diff
@@ -0,0 +1,2 @@
+class ShowApiAction < ApiAction
+end
```

Comment:
@Pralish same here for ShowApiAction and UpdateApiAction -- are they being used?